test/kubemark/resources/manifests/kube-addon-manager.yaml

```diff
@@ -12,8 +12,7 @@ spec:
     # When updating version also bump it in:
     # - cluster/images/hyperkube/static-pods/addon-manager-singlenode.json
     # - cluster/images/hyperkube/static-pods/addon-manager-multinode.json
-    # - cluster/gce/coreos/kube-manifests/kube-addon-manager.yaml
```

Comment:
The path for `kube-addon-manager.yaml` was actually updated to

```
cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml
test/kubemark/resources/manifests/kube-addon-manager.yaml
```

My personal thinking is that we can probably remove all of the comments here 

```
    # When updating version also bump it in:
    # - cluster/images/hyperkube/static-pods/addon-manager-singlenode.json
    # - cluster/images/hyperkube/static-pods/addon-manager-multinode.json
    # - cluster/gce/coreos/kube-manifests/kube-addon-manager.yaml
```

As the comments is difficult to maintain and also I think that most developer can simply `grep` to get and replace all needed places.